hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/replication/ContainerReplicaCount.java

```diff
@@ -37,7 +38,8 @@ public interface ContainerReplicaCount {
 
   boolean isSufficientlyReplicated();
 
-  boolean isSufficientlyReplicatedForOffline(DatanodeDetails datanode);
+  boolean isSufficientlyReplicatedForOffline(DatanodeDetails datanode,
```

Comment:
Maybe we should add some Java doc here to explain what this method is for, or down at the implementation level if the description is going to be different for Legacy, EC and Ratis Counts?

Reply:
Sure, I'll add javadoc once we've concluded our discussion above about `isHealthyEnoughForOffline` and `isSufficientlyReplicatedForOffline`.